[TIMOB-24250] Android: TextField AUTOCAPITALIZATION_ALL property doesn't work with googles new keyboard: GBoard

maggieaxway opened the pull request:
Titanium.UI.TEXT_AUTOCAPITALIZATION_ALL is not working. It doesn't work in native Android as well if user choose shift to enter small letter. Listen to the TextChange to fix the bug

**JIRA:** https://jira.appcelerator.org/browse/TIMOB-24250

maggieaxway committed:
Update TiUIText.java

Titanium.UI.TEXT_AUTOCAPITALIZATION_ALL is not working. It doesn't work in native Android as well if user choose shift to enter small letter. Listen to the TextChange to fix the bug

garymathews commented on android/modules/ui/src/java/ti/modules/titanium/ui/widget/TiUIText.java:
```diff
@@ -84,6 +84,8 @@
 
 	protected EditText tv;
+	
+	private KrollDict keyboard;
```
```Java
private boolean autoCapAll = false;
```

garymathews commented on android/modules/ui/src/java/ti/modules/titanium/ui/widget/TiUIText.java:
```diff
@@ -472,6 +493,9 @@ public void handleKeyboard(KrollDict d)
+		
 		if (d.containsKey(TiC.PROPERTY_AUTOCORRECT) && !TiConvert.toBoolean(d, TiC.PROPERTY_AUTOCORRECT, true)) {
 			autocorrect = 0;
 		}
```
##### LINE 548
```Java
...
case TEXT_AUTOCAPITALIZATION_ALL:
    autoCapValue = InputType.TYPE_TEXT_FLAG_CAP_CHARACTERS;
    autoCapAll = true;
    break;
```

set `autoCapAll = false;` for other AUTOCAPITALIZATION conditions

maggieaxway committed:
add autoCapAll flag

add autoCapAll flag to indicate when TEXT_AUTOCAPITALIZATION_ALL is ON and to capitalize manually

maggieaxway committed:
add InputFilter to screen small letter input and remove unnecessary InputType constants

garymathews commented on android/modules/ui/src/java/ti/modules/titanium/ui/widget/TiUIText.java:
```diff
@@ -8,6 +8,7 @@
 
 import java.util.HashMap;
 
+import android.text.*;
```
Revert import changes, we try not to use wildcards
